Unable to import EOS Samples

**Describe the bug**
After installing the plugin built off develop, importing the plugin and the samples throws the following error:
`
Assets\Samples\Epic Online Services Plugin for Unity\2.1.7\EOS Plugin for Unity Samples\Scripts\UI\UIP2PTransportMenu.cs(59,17): error CS0246: The type or namespace name 'EOSTransportManager' could not be found (are you missing a using directive or an assembly reference?)
`
It also seems to be unable to find NetworkSamplePlayer.

**To Reproduce**
Steps to reproduce the behavior:
1. Build the EOS Unity plugin on Develop
2. Create a new project
3. Import the EOS Plugin tarball
4. Import the EOS Samples
5. The error should pop up after import

**Expected behavior**
No exception on importing the plugin samples

**Desktop (please complete the following information):**
 - OS: Windows 10 x64
 - Unity Version: 2022.2.0b16
 - Plugin Version: develop

**Additional context**
Seems the networking folder's scripts might just not be brought over, maybe a couple more entries in the package description configs are needed?

--- a/PackageDescriptionConfigs/eos_package_description.json
+++ b/PackageDescriptionConfigs/eos_package_description.json
@@ -79,16 +79,16 @@
         {"ignore_regex":"Assets/Plugins/Windows/.*/steam_api64\\.dll"},
 
         {"comment": "----------------------------End Windows---------------------------------------------"},
-		
-		{"src":"Assets/Plugins/Linux.meta", "dest": "Runtime/"},
-		
-		{"src":"Assets/Plugins/Linux/license.txt", "dest": "Runtime/Linux/"},
+
+        {"src":"Assets/Plugins/Linux.meta", "dest": "Runtime/"},
+        
+        {"src":"Assets/Plugins/Linux/license.txt", "dest": "Runtime/Linux/"},
         {"src":"Assets/Plugins/Linux/license.txt.meta", "dest": "Runtime/Linux/"},
         {"src":"Assets/Plugins/Linux/ThirdPartySoftwareNotice.txt", "dest": "Runtime/Linux/"},
         {"src":"Assets/Plugins/Linux/ThirdPartySoftwareNotice.txt.meta", "dest": "Runtime/Linux/"},
-		{"src":"Assets/Plugins/Linux/*.so", "dest": "Runtime/Linux/"},
+        {"src":"Assets/Plugins/Linux/*.so", "dest": "Runtime/Linux/"},
         {"src":"Assets/Plugins/Linux/*.so.meta", "dest": "Runtime/Linux/"},
-		{"src":"Assets/Plugins/Linux/*.cs", "dest": "Runtime/Linux/"},
+        {"src":"Assets/Plugins/Linux/*.cs", "dest": "Runtime/Linux/"},
         {"src":"Assets/Plugins/Linux/*.cs.meta", "dest": "Runtime/Linux/"},
 
         {"src":"Assets/Plugins/Source/Editor/com.playeveryware.eos-Editor.asmdef", "dest": "Editor/"},
@@ -96,9 +96,9 @@
 
         {"src":"Assets/Plugins/Source/Editor/*.cs", "dest": "Editor/"},
         {"src":"Assets/Plugins/Source/Editor/*.cs.meta", "dest": "Editor/"},
-		
-		{"src":"Assets/Plugins/Source/Editor/Linux.meta", "dest": "Editor/"},
-		{"src":"Assets/Plugins/Source/Editor/Linux/*.cs", "dest": "Editor/Linux/"},
+
+        {"src":"Assets/Plugins/Source/Editor/Linux.meta", "dest": "Editor/"},
+        {"src":"Assets/Plugins/Source/Editor/Linux/*.cs", "dest": "Editor/Linux/"},
         {"src":"Assets/Plugins/Source/Editor/Linux/*.cs.meta", "dest": "Editor/Linux/"},
 
         {"comment": "----------------------------macOS---------------------------------------------"},
@@ -166,8 +166,8 @@
         {"src":"Assets/Plugins/Source/EOS_SDK/Generated/Connect.meta", "dest": "Runtime/EOS_SDK/Generated/"},
         {"src":"Assets/Plugins/Source/EOS_SDK/Generated/Connect/*.cs", "dest": "Runtime/EOS_SDK/Generated/Connect/"},
         {"src":"Assets/Plugins/Source/EOS_SDK/Generated/Connect/*.cs.meta", "dest": "Runtime/EOS_SDK/Generated/Connect/"},
-		
-		{"src":"Assets/Plugins/Source/EOS_SDK/Generated/CustomInvites.meta", "dest": "Runtime/EOS_SDK/Generated/"},
+
+        {"src":"Assets/Plugins/Source/EOS_SDK/Generated/CustomInvites.meta", "dest": "Runtime/EOS_SDK/Generated/"},
         {"src":"Assets/Plugins/Source/EOS_SDK/Generated/CustomInvites/*.cs", "dest": "Runtime/EOS_SDK/Generated/CustomInvites/"},
         {"src":"Assets/Plugins/Source/EOS_SDK/Generated/CustomInvites/*.cs.meta", "dest": "Runtime/EOS_SDK/Generated/CustomInvites/"},
 
@@ -222,8 +222,8 @@
         {"src":"Assets/Plugins/Source/EOS_SDK/Generated/Presence.meta", "dest": "Runtime/EOS_SDK/Generated/"},
         {"src":"Assets/Plugins/Source/EOS_SDK/Generated/Presence/*.cs", "dest": "Runtime/EOS_SDK/Generated/Presence/"},
         {"src":"Assets/Plugins/Source/EOS_SDK/Generated/Presence/*.cs.meta", "dest": "Runtime/EOS_SDK/Generated/Presence/"},
-		
-		{"src":"Assets/Plugins/Source/EOS_SDK/Generated/ProgressionSnapshot.meta", "dest": "Runtime/EOS_SDK/Generated/"},
+
+        {"src":"Assets/Plugins/Source/EOS_SDK/Generated/ProgressionSnapshot.meta", "dest": "Runtime/EOS_SDK/Generated/"},
         {"src":"Assets/Plugins/Source/EOS_SDK/Generated/ProgressionSnapshot/*.cs", "dest": "Runtime/EOS_SDK/Generated/ProgressionSnapshot/"},
         {"src":"Assets/Plugins/Source/EOS_SDK/Generated/ProgressionSnapshot/*.cs.meta", "dest": "Runtime/EOS_SDK/Generated/ProgressionSnapshot/"},
 
@@ -358,14 +358,14 @@
         {"src":"Assets/Scripts/UI/*.cs.meta", "dest": "Samples~/Samples/Scripts/UI/"},
         {"src":"Assets/Scripts/Util/*.cs", "dest": "Samples~/Samples/Scripts/Util/"},
         {"src":"Assets/Scripts/Util/*.cs.meta", "dest": "Samples~/Samples/Scripts/Util/"},
-		{"src":"Assets/Scripts/Networking/*.cs", "dest": "Samples~/Samples/Scripts/Networking/"},
+        {"src":"Assets/Scripts/Networking/*.cs", "dest": "Samples~/Samples/Scripts/Networking/"},
         {"src":"Assets/Scripts/Networking/*.cs.meta", "dest": "Samples~/Samples/Scripts/Networking/"},
 
         {"src":"Assets/Scenes.meta", "dest": "Samples~/Samples/"},
         {"src":"Assets/Scenes/*.unity", "dest": "Samples~/Samples/Scenes/"},
         {"src":"Assets/Scenes/*.unity.meta", "dest": "Samples~/Samples/Scenes/"},
-		
-		{"src":"Assets/Readme.meta", "dest": "Samples~/Samples/"},
+
+        {"src":"Assets/Readme.meta", "dest": "Samples~/Samples/"},
         {"src":"Assets/Readme/*.txt", "dest": "Samples~/Samples/Readme/"},
         {"src":"Assets/Readme/*.txt.meta", "dest": "Samples~/Samples/Readme/"}
     ],

--- a/PackageDescriptionConfigs/eos_package_description.json
+++ b/PackageDescriptionConfigs/eos_package_description.json
@@ -358,6 +358,8 @@
         {"src":"Assets/Scripts/UI/*.cs.meta", "dest": "Samples~/Samples/Scripts/UI/"},
         {"src":"Assets/Scripts/Util/*.cs", "dest": "Samples~/Samples/Scripts/Util/"},
         {"src":"Assets/Scripts/Util/*.cs.meta", "dest": "Samples~/Samples/Scripts/Util/"},
+		{"src":"Assets/Scripts/Networking/*.cs", "dest": "Samples~/Samples/Scripts/Networking/"},
+        {"src":"Assets/Scripts/Networking/*.cs.meta", "dest": "Samples~/Samples/Scripts/Networking/"},
 
         {"src":"Assets/Scenes.meta", "dest": "Samples~/Samples/"},
         {"src":"Assets/Scenes/*.unity", "dest": "Samples~/Samples/Scenes/"},

